src/components/CarDetails.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { CircularProgress, Typography } from '@mui/material';
+import { fetchCarById } from '../redux/actions/carActions';
+
+const CarDetails = () => {
```

Comment:
- [ ] When I try to get to any car details I get this error:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/ed297bbd-bd47-44a4-9cdc-62ba63ec8ed0)

Since the server responds with a 200 and the data that is being retrieved is correct, the bug should be in the frontend.

Server response:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/6181a162-64fe-41f8-8d69-dc743be32ce2)


Kindly fix this bug so your app works as intended. 👍